File: backend/src/main/java/com/demo/student_management/dto/admin/PhanCongResponse.java
package com.demo.student_management.dto.admin;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class PhanCongResponse {
    private String idPhanCong;
    private String idGiaoVien;
    private String tenGiaoVien;
    private String idLop;
    private String tenLop;
    private String idMonHoc;
    private String tenMonHoc;
    private String idHocKy;
    private String tenHocKy;
}

File: frontend/StudentManagement.Shared/Dtos/Admin/PhanCongResponse.cs
namespace StudentManagement.Shared.Dtos.Admin;

public sealed class PhanCongResponse
{
    public string IdPhanCong { get; set; } = string.Empty;
    public string IdGiaoVien { get; set; } = string.Empty;
    public string TenGiaoVien { get; set; } = string.Empty;
    public string IdLop { get; set; } = string.Empty;
    public string TenLop { get; set; } = string.Empty;
    public string IdMonHoc { get; set; } = string.Empty;
    public string TenMonHoc { get; set; } = string.Empty;
    public string IdHocKy { get; set; } = string.Empty;
    public string TenHocKy { get; set; } = string.Empty;
}
